app/routes/blog/index.tsx

```diff
@@ -33,22 +44,102 @@ export function meta() {
   }
 }
 
+const fakePostListItems = Array.from({length: 3})
+  .flatMap(() => articles)
+  .map((article, idx) => ({...article, articleUrl: `/blog-${idx}`}))
+
 function BlogHome() {
-  const posts = useRouteData<Array<PostListItem>>()
+  // TODO: use the real data. I used fixtures here, because real data was missing readTime, imageAlt, and only had 2 entries.
+  const [featured, ...posts] = fakePostListItems // useRouteData<Array<PostListItem>>()
+
   return (
     <div>
-      <header>
-        <h2>Kent C. Dodds</h2>
-      </header>
-      <main>
-        {posts.map(post => (
-          <p key={post.slug}>
-            <Link to={`/blog/${post.slug}`}>{post.frontmatter.meta.title}</Link>
-            <br />
-            <small>{post.frontmatter.meta.description}</small>
-          </p>
+      <Grid className="grid-rows-max-content mt-16">
+        <div className="col-span-full lg:col-span-6 lg:col-start-7 lg:row-span-2">
+          <img
+            className="max-h-[50vh] mx-auto lg:max-w-md"
+            src={skis.src}
+            alt={skis.alt}
+          />
+        </div>
+
+        <div className="col-span-full lg:col-span-6 lg:row-start-1">
+          <div className="space-y-2 lg:max-w-sm">
+            <H2>Learn development with great articles.</H2>
+            <H2 variant="secondary" as="p">
+              Find the latest of my writing here.
+            </H2>
+          </div>
+        </div>
+
+        <form className="col-span-4 mt-6 lg:row-start-2">
+          <div className="relative">
+            <div className="absolute left-8 top-0 flex items-center justify-center h-full text-blueGray-500">
+              <SearchIcon />
+            </div>
+            <input
+              placeholder="Search blog"
+              aria-label="Search blog"
+              className="dark:focus:bg-gray-800 placeholder-black dark:placeholder-white px-16 py-6 w-full text-black dark:text-white text-lg font-medium focus:bg-gray-100 bg-transparent border border-gray-200 dark:border-gray-600 rounded-full focus:outline-none"
+            />
+            <div className="absolute right-8 top-0 flex items-center justify-center h-full text-blueGray-500 text-lg font-medium">
+              {/* TODO: replace count */}
+              173
+            </div>
+          </div>
+        </form>
+      </Grid>
+
+      <Spacer size="small" />
+
+      <Grid>
+        <div className="col-span-full">
+          <H6 as="p">Search blog by topics</H6>
+        </div>
+        <div className="flex flex-wrap col-span-full -ml-4 -mt-4 lg:col-span-10">
+          {/* TODO: get tags from database, and connect `selected` to something */}
+          {tags.map((tag, idx) => (
+            <Tag key={tag} tag={tag} selected={idx === 3} />
+          ))}
+        </div>
+      </Grid>
+
+      {/* TODO: this should be `smallest`, but `medium` looks better? */}
```

Comment:
Not sure if I made the right call here. I think that `medium` looks better. But please compare with `smallest` before merging.